### PR TITLE
HEV Suit Balance

### PR DESCRIPTION
**Pull request by Gandalf2k15**
HEV suits are now space worthy. Have no slowdown like they had before. 
Fixes https://github.com/HippieStationCode/HippieStation13/issues/2564
Map changes are for new directorys.

### CODE REVIEW

**Comment by ArcLumin** [comment]
Not really a fix though, as it wasn't a bug? Pretty sure they were never meant to be spaceworthy.

I'm fine with this, however.

**Comment by TDcoolguy300** [comment]
Not a bug fix

**Comment by ghost** [comment]
This isn't a bugfix.

**Comment by Voratius** [comment]
Hey guys, I don't think this is a bug fix.

**Comment by Gandalf2k15** [comment]
Shit it's not a bug fix

File: code/modules/clothing/spacesuits/miscellaneous.dm [hunk 1]
```diff
@@ -357,6 +358,32 @@ Contains:
...
+	name = "H.E.V Suit"
+	desc = "A highly advanced Hazardous Environment suit, this is the Mk7 version. Able to efficiently protect the user from just about any danger that a true scientist in the field may encounter. Also has some moderate protection from lasers, bullets, and blunt objects, just incase an angry military force decides to attempt to stop the progression of SCIENCE!"
+	icon_state = "hev"
+	item_state = "armor"
+	w_class = 4		//bulky item
+	strip_delay = 120
+	put_on_delay = 70
+	unacidable = 1	//Acids have no effect
+	burn_state = -1 //Not Burnable
+	slowdown = 0 //Fast!
```
Comment (TDcoolguy300) [inline code/modules/clothing/spacesuits/miscellaneous.dm]: This was not mentioned in your damn PR

**Comment by Nebyab** [comment]
not a bugfix

and I don't really care either way tbh

**Comment by Spacedong** [comment]
yeah sure

**Comment by McHatters** [comment]
yes

**Comment by ghost** [comment]
A fully spaceworthy armor with decent protection with no slowdown?
That's pretty absurd.

**Comment by dreamcrab** [comment]
guys this isnt a bugfix

but its good +half-life 3

**Comment by Spacedong** [comment]
nebyab doesn't care and dreamcrab likes it so /shrug

**Comment by JohnGinnane** [comment]
Code looks good

**Comment by JohnGinnane** [comment]
@Nebyab is that your approval?

**Comment by Nebyab** [comment]
its good enough approval wise i guess

i give 0 shits either way, so merge it or close it

**Comment by TDcoolguy300** [comment]
I like to express my disapproval of the HEV being spaceworthy in this PR, if thats worth it.

**Comment by ghost** [comment]
Neb and teamstab want it so I'm outruled.

**Comment by ArcLumin** [comment]
I feel like there should be makable spacesuits, maybe with some sort of drawback. However, this doesn't have a drawback, and is armored. I guess you have to do a tiny bit of RnD for it though... Dunno exactly if I like this or not.

**Comment by Nebyab** [comment]
i've found a reason to care

nah

**Comment by TDcoolguy300** [comment]
Denied

**Pull request closed without merging**